documentation/modules/ref-console-address-filtering.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// master.adoc
+
+[id='ref-console-address-filtering-{context}']
+= Using the {ProductName} Console address filtering
+
+The address name filtering feature in the {ProductName} Console uses regular expressions. Also, filters are cumulative.
+
+.{ProductName} Console address name filtering behavior
+[cols="34%a,33%a,33%a",options="header"]
+|===
+|To match... |Use this |Result
```

Comment:
Change 'use this' to 'use' and remove use from the options below.  And maybe 'Result' should be 'Returns' or 'Results in' ... think of the headings as part of the sentence.

Reply:
Done